### DISCUSSION TITLE
about Block

### DISCUSSION THREAD

**Discussion by leo-brown244** [Cyfrin/foundry-full-course-cu#1977] [General]
in lesson 11, block consist of block, nonce, data.
I want to know more detail about this concepts..

**Comment by EngrPips** [comment]
Hello @leo-brown244, here `https://www.geeksforgeeks.org/ethereum-block-structure/`

**Reply by leo-brown244** [reply]
thanks a lot~

**Reply by EngrPips** [reply]
You are welcome.
